FiberSpectrograph/v4/_summit.yaml

```diff
@@ -0,0 +1,3 @@
+s3instance: cp
+image_service_url: "http://ccs.lsst.org"
+location: "auxtel"
```

Comment:
The double quotes aren't needed and I suggest omitting them. (They might be wanted for URLs because of the contained ":" so best to leave them for that).

Same comment for _tucson.yaml

Reply:
So single quotes? or no quotes needed in this format?

Reply:
No quotes are needed. (I'm not even sure if yaml supports single quotes; I never use them in yaml files).